testing_and_setup/compass/ocean/hurricane/init_template.xml

```diff
@@ -9,6 +9,7 @@
 		<option name="config_hurricane_vert_levels">100</option>
 		<option name="config_hurricane_min_depth">10.0</option>
 		<option name="config_hurricane_max_depth">6000.0</option>
+		<option name="config_use_variable_drag">.true.</option>
```

Comment:
You could add the other limit/drag parameters here so that the defaults could be more easily changed. Maybe this would be more appropriate in the case's `config_init.xml` file.

Reply:
It is probably appropriate here to be clear as to what is happening by adding them -- great suggestion!

Reply:
Please see below, thanks!